src/main/java/seedu/tasklist/ui/HelpWindow.java

```diff
@@ -20,30 +20,35 @@
             + " try:\n\n"
             + "-----------------------------------------Basic Features---------------------------------------------\n"
             + "- To add an assignment: add n/Lab report 3 d/23-04-2020 1230 mod/CS2100\n"
-            + " *Tip: You can include `remind` when adding an assignment for it to appear in Your reminders\n"
+            + "    *Tip: You can include `remind` when adding an assignment for it to appear in Your reminders\n"
             + "- To delete an assignment shown in the list: delete 3\n"
-            + " *Tip: You can delete multiple assignments by including multiple indexes: delete 3 4 7\n"
+            + "    *Tip: You can delete multiple assignments by including multiple indexes: delete 3 4 7\n"
             + "- To list all assignments: list\n"
-            + " *Tip: Use `list 2` to list assignments with deadlines 2 days from current date\n\n"
+            + "    *Tip: Use `list 2` to list assignments with deadlines 2 days from current date\n\n"
             + "----------------------------------------Advance Features-------------------------------------------\n"
             + "- To import your timetable: import url/YOUR_NUSMODS_URL\n"
-            + "- To undo the most recent command: undo\n"
+            + "- To undo your most recent command: undo\n"
             + "- To find assignments by name: find n/Lab\n"
-            + " *Tip: You can also find assignments by module code, deadline or priority using\n"
-            + " `mod/`, `d/` and `p/` prefixes respectively.\n"
+            + "    *Tip: You can also find assignments by module code, deadline or priority using\n"
+            + "    `mod/`, `d/` and `p/` prefixes respectively.\n"
+            + "- To edit the details of an assignment: edit 3 n/Lab 5\n"
+            + "    *Tip: You can edit multiple fields, such as name, deadline and module code, "
+            + "     of an assignment at the same using `n/`, `d/` and `mod/` prefixes respectively.\n"
+            + "- To schedule an assignment: schedule 3 hrs/2 by/10-11-2020 2359\n"
+            + "- To unschedule an assignment: unschedule 3\n"
```

Comment:
Has @minhhhnguyen2000 been informed of the change in prefixes for the schedule command?

Reply:
I changed it based on the prefixes he mentioned during our call that day. @minhhhnguyen2000 pls take note 👍

Reply:
@printinghelloworld I just updated schedule feature to now have "do after" feature, The new format is:
schedule INDEX hrs/EXPECTED_HOUR af/AFTER_DATE AFTER_TIME by/BEFORE_DATE BEFORE_TIME